src/StreamContextFactory.php

```diff
@@ -47,9 +54,9 @@ public function create(ConfigInterface $config)
         }
 
         if (count($headers) > 0) {
-            $options['http']['header'] = $headers;
+            $streamContextOptions['http']['header'] = $headers;
         }
 
-        return stream_context_create($options);
+        return stream_context_create($streamContextOptions);
```

Comment:
As I see it this applies the stream context options during the code generation phase. We normally also [apply the same configuration to the generated Service class](https://github.com/wsdl2phpgenerator/wsdl2phpgenerator/blob/master/src/Service.php#L165) so the same options apply during the SOAP service call phase.

Could we do this here as well?